### PR TITLE
Added italian translation

### LINKED ISSUE #16: Italian (Italiano) <it-it.cfg>
# Localization - ![Italiano][IT] Italian (Italiano) <it-it.cfg>

## GitHub

* [x] assign
  * [x] zer0Kerbal
  * [x] Repo Localization Project
  * [x] Label: localization
  * [x] Label: help-wanted
  * [x] Milestone: Localization - Master

### GameData

* [x] add  Localization\[it-it.cfg] 🗃️

### Translation

* [x] ![Italiano][IT] Italian (Italiano) <it-it.cfg>

### Documentation

* [x] Update [readme.md] 🔢
* [x] Update [release.md] 🧾

## Wiki

* [x] Update docs/[localization.md]

### Update Social Media

* [x] GitHub (automated) (zer0Kerbal/zer0Kerbal)
* [x] CurseForge
* [x] Reddit
* [x] Twitter @zer0Kerbal
* [x] Forum
* [x] Patreon
* [x] SpaceDock
* [x] CKAN (automated)

[lclztn]: https://github.com/zer0Kerbal/zer0Kerbal/blob/master/Localization/readme.md  "Localization"
[qs]: https://github.com/zer0Kerbal/zer0Kerbal/blob/master/Localization/quickstart.md "Quick Start"

[IT]: https://raw.githubusercontent.com/zer0Kerbal/zer0Kerbal/master/img/IT.png "Italiano"

this file: © All Rights Reserved by zer0Kerbal

### PR DESCRIPTION
**Pull request by MattNot**
I translated "Omni" as "Universal", in Italian there isn't a real difference between the two terms as "omnia" is the latin for "everything". Let me know if is okay for you. Can you please add this repo to hacktoberfest? Ty.

closes #16

### CODE REVIEW

**Comment by zer0Kerbal** [comment]
closes #16

**Comment by zer0Kerbal** [comment]
> I translated "Omni" as "Universal", in Italian there isn't a real difference between the two terms as "omnia" is the latin for "everything". Let me know if is okay for you. Can you please add this repo to hacktoberfest? Ty.
> 
> closes #16

@MattNot either way would be fine - although Omnia might be better because of the addon's abbreviation is `ROS`.

hacktoberfest - I believe all I have to do it add `hacktoberfest` to the repo tags; if so that is done.

Thank you!

File: GameData/nli2work/RadialOmniSeparator/Localization/it-it.cfg [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// en-us.cfg v 1.0.0.0
+// Radial Omni Separator (ROS)
+// created: 05 Oct 21
+// updated: 30 Sept 2022
+
```
Comment (zer0Kerbal) [inline GameData/nli2work/RadialOmniSeparator/Localization/it-it.cfg:5]: `// translated by @MattNot `

**Review by zer0Kerbal — APPROVED** [review]
hacktoberfest approved.

see two comments, will give chance to update if so choose, otherwise will merge. 

Thank you!
zed'K

**Comment by MattNot** [comment]
Hi! Yeah, maybe leaving "Omnia" is better. I'll do this asap I'll return home from work. For hacktoberfest, the label should be just "hacktoberfest" (without "accepted") otherwise it won't count, thank you

**Comment by zer0Kerbal** [comment]
> Hi! Yeah, maybe leaving "Omnia" is better. I'll do this asap I'll return home from work. For hacktoberfest, the label should be just "hacktoberfest" (without "accepted") otherwise it won't count, thank you

I want your contributions to be counted so I looked up https://hacktoberfest.com/participation/

I could be wrong so please confirm

```
HERE’S HOW WE VALIDATE CONTRIBUTOR PULL/MERGE REQUESTS (“PR/MRS”) FOR HACKTOBERFEST
...
YOUR PR/MRS MUST BE IN A REPO TAGGED WITH THE “HACKTOBERFEST” TOPIC, OR BE LABELED “HACKTOBERFEST-ACCEPTED.”
...
```

**Comment by MattNot** [comment]
Chenged the term, hacktoberfest is okay. Ty :-)

**Review by zer0Kerbal — COMMENTED** [review]
🚀

**Comment by zer0Kerbal** [comment]
> Changed the term, hacktoberfest is okay. Ty :-)

thank you!

**Pull request merged**